salt/modules/panos.py

```diff
@@ -0,0 +1,1854 @@
+# -*- coding: utf-8 -*-
+"""
+Module to provide Palo Alto compatibility to Salt.
+
+:codeauthor: :email:`Spencer Ervin <spencer_ervin@hotmail.com>`
+:maturity:   new
+:depends:    none
+:platform:   unix
+
+
+Configuration
+=============
+This module accepts connection configuration details either as
+parameters, or as configuration settings in pillar as a Salt proxy.
+Options passed into opts will be ignored if options are passed into pillar.
+
+.. seealso::
+    :prox:`Palo Alto Proxy Module <salt.proxy.panos>`
+
+About
+=====
+This execution module was designed to handle connections to a Palo Alto based
+firewall. This module adds support to send connections directly to the device
+through the XML API or through a brokered connection to Panorama.
+
+"""
+
+# Import Python Libs
+from __future__ import absolute_import
+import logging
+import time
+
+# Import Salt Libs
+import salt.proxy.panos
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'panos'
+
+
+def __virtual__():
+    """
+    Will load for the panos proxy minions.
+    """
+    try:
+        if salt.utils.is_proxy() and \
```

Comment:
Same comment here about the moved function. This should be `salt.utils.platform.is_proxy()`. And also please include the `import salt.utils.platform` in the `# Import Salt Libs` section above.